Week 2 - Introduction to Java/Homework/README.md

```diff
@@ -0,0 +1,26 @@
+# Bài 1: 
+In ra màn hình dòng chữ “Hello, this is my first appliation”.
+# Bài 2: 
+Nhập 1 số nguyên n từ bàn phím và hiển thị giá trị vừa nhập ra màn hình.
```

Comment:
Mình khuyên trong file này những biến như `n`, `n1`, `m1`, `m2`, `max`, `min`, công thức như `ax+b` thì đều nên bỏ vào code nha

Consistency is key

Reply:
okay